font/src/lib.rs

```diff
@@ -267,3 +321,27 @@ pub trait Rasterize {
     /// Update the Rasterizer's DPI factor
     fn update_dpr(&mut self, device_pixel_ratio: f32);
 }
+
+/// Config option specific to the Rasterizer.
+/// Since the Rasterizer lives in the subcrate font we do not want it to depend on the Font config
+/// struct from alacritty, as this would introduce a circular dependency between the crates.Clone
+/// This struct specifies the subset of Font that the Rasterizer cares about, then traits can be
+/// used to convert from Font to this struct when constructing a Rasterizer.
```

Comment:
I don't think we need this comment. It should be understood that crates should not depend on each other.

Rather than documenting this, the doc comment for this configuration should describe what it is used for.

> Config option specific to the Rasterizer.

I think the wording here specifically is already a bit odd. It's not a `config option`, it's the configuration for the rasterizer. Crates in the workspace should all be treated largely independently, so please comment this in a way you would for a crate completely unrelated to Alacritty.